app/services/idv/steps/verify_step.rb

```diff
@@ -14,6 +14,23 @@ def enqueue_job
 
         pii_from_doc[:uuid_prefix] = ServiceProvider.find_by(issuer: sp_session[:issuer])&.app_id
 
+        if pii_from_doc[:ssn].present?
+          throttle = Throttle.for(
+            target: Digest::SHA256.hexdigest(pii_from_doc[:ssn]),
+            throttle_type: :proof_ssn,
+          )
+
+          if throttle.throttled_else_increment?
```

Comment:
Another option would be to override `idv_throttle_params` from the base class: https://github.com/18F/identity-idp/blob/main/app/services/idv/steps/doc_auth_base_step.rb#L10-L15 `

My thinking is, that rate limiting helps to limit individual users doing IDV too much, so by making this a separate check, we still want to have both throttles